src/v/cloud_storage_clients/abs_client.cc

```diff
@@ -826,7 +876,66 @@ ss::future<abs_client::list_bucket_result> abs_client::do_list_objects(
 
 ss::future<result<abs_client::storage_account_info, error_outcome>>
 abs_client::get_account_info(ss::lowres_clock::duration timeout) {
```

Comment:
I'd rename this to `check_hns_active`. We don't really get the account info anymore. The naming is confusing. Imagine the frustraction of someone in the future trying to use this thinking that you really get account info in there

Reply:
alternative: https://github.com/redpanda-data/redpanda/pull/17840#discussion_r1570441764